receiver/mongodbreceiver/scraper_test.go

```diff
@@ -57,65 +57,6 @@ func TestScraperLifecycle(t *testing.T) {
-				"failed to collect metric mongodb.extent.count with attribute(s) fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.index.size with attribute(s) fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.index.count with attribute(s) fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.object.count with attribute(s) fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.storage.size with attribute(s) fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.connection.count with attribute(s) available, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.connection.count with attribute(s) current, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.connection.count with attribute(s) active, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.document.operation.count with attribute(s) inserted, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.document.operation.count with attribute(s) updated, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.document.operation.count with attribute(s) deleted, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.memory.usage with attribute(s) resident, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.memory.usage with attribute(s) virtual, fakedatabase: could not find key for metric",
-				"failed to collect metric mongodb.index.access.count with attribute(s) fakedatabase, orders: could not find key for index access metric",
-				"failed to collect metric mongodb.index.access.count with attribute(s) fakedatabase, products: could not find key for index access metric",
-			}, "; "))
-	errAllClientFailedFetch = errors.New(
-		strings.Join(
-			[]string{
-				"failed to fetch admin server status metrics: some admin server status error",
-				"failed to fetch top stats metrics: some top stats error",
-				"failed to fetch database stats metrics: some database stats error",
-				"failed to fetch server status metrics: some server status error",
-				"failed to fetch index stats metrics: some index stats error",
-				"failed to fetch index stats metrics: some index stats error",
-			}, "; "))
-
-	errCollectionNames = errors.New(
-		strings.Join(
-			[]string{
-				"failed to fetch admin server status metrics: some admin server status error",
-				"failed to fetch top stats metrics: some top stats error",
-				"failed to fetch database stats metrics: some database stats error",
-				"failed to fetch server status metrics: some server status error",
-				"failed to fetch collection names: some collection names error",
-			}, "; "))
-)
-
```

Comment:
Let's please move these back here since it appears this change was unnecessary for the pr.